tests/rptest/tests/restart_services_test.py

```diff
@@ -61,9 +61,17 @@ def test_restart_services(self):
             self.logger.debug(ex)
             assert ex.response.status_code == requests.codes.not_found
 
+        # Success checks
         self.logger.debug("Check schema registry restart")
         result_raw = admin.redpanda_services_restart(
             rp_service='schema-registry')
         check_service_restart(self.redpanda, "Restarting the schema registry")
         self.logger.debug(result_raw)
         assert result_raw.status_code == requests.codes.ok
+
+        self.logger.debug("Check metadata dissemination service restart")
+        result_raw = admin.redpanda_services_restart(
+            rp_service='metadata-dissemination-service')
+        check_service_restart(self.redpanda, "Restarting the metadata dissemination service")
+        self.logger.debug(result_raw)
+        assert result_raw.status_code == requests.codes.ok
```

Comment:
@abhijat Any recommendation on a better test for the metadata dissemination service? I was thinking of a test to check that leadership remains the same after restart.